test/run-unittest.d

```diff
@@ -46,12 +47,15 @@ int main(string[] args)
 	//** DC_BIN=$(basename "$DC")
 	//** CURR_DIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
 	//** FRONTEND="${FRONTEND:-}"
-	const dc_bin = baseName(dc);
+	const dc_bin = baseName(dc).stripExtension;
 	const curr_dir = __FILE_FULL_PATH__.dirName();
-	const frontend = environment.get("FRONTEND", "");
+	const frontend = environment.get("FRONTEND", __VERSION__.format!"%04d");
```

Comment:
I assume that is to make it easier to test with the host compiler ?

Reply:
Yes. That is correct. This is useful for manually redoing failed tests. Only DUB variables are required to run.
`dub --single run-unittest.d -- xxxx`